### PR TITLE
Cleanup go.mod

### PR DESCRIPTION
**Pull request by 89z**
Bunch of indirect modules, that are removed when running `go mod tidy`

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,18 +1,12 @@
 module github.com/walles/moar
 
-go 1.13
+go 1.16
```
Comment (walles) [inline go.mod:3]: Can this be kept at `1.13`?

**Comment by walles** [comment]
Tried to push a fix to this branch but wasn't allowed, added this change to `master` to catch this in CI next time: 97f5233080ce42c4440573091749148544a83e74

Please rebase on latest master and do...
```
go get "honnef.co/go/tools/cmd/staticcheck@2020.2.3" "github.com/kisielk/errcheck@v1.6.0"
```

... then commit `go.mod` changes.

Not sure whether you can run shell scripts, in that case running `./test.sh` will work as well.

Thing is that if the linters aren't already in `go.mod`, then `go.mod` will be modified each time `./test.sh` is run, which makes local development impractical.

**Comment by walles** [comment]
Or maybe just re-cloning `master` and running `go mod tidy` would be easier, conflicts are the horrors.

**Comment by 89z** [comment]
Ok I switched back to Go 1.13

**Comment by walles** [comment]
The build...

https://travis-ci.com/github/walles/moar/jobs/500210774

... is failing with the following message at the end of the build log:

```
ERROR: go.mod modified by installing linters, run './test.sh' locally or this, commit your changes and try again:
$ go get honnef.co/go/tools/cmd/staticcheck@2020.2.3 github.com/kisielk/errcheck@v1.6.0
```

**Comment by 89z** [comment]
You keep making commits, so I keep having to rebase against master. I dont know if thats the problem, or if the CI is doing some bullshit. Maybe try again with my latest commit?

**Comment by 89z** [comment]
Ok I think I fixed it. You are using `go get` in the `test.sh` file, when it should be `go install`, as those modules are only used in the CI setting.

**Pull request closed without merging**